app/views/statuses/_form.html.erb

```diff
@@ -45,11 +45,9 @@ See COPYRIGHT and LICENSE files for more details.
   <div class="form--field">
     <% disabled = !EnterpriseToken.allows_to?(:readonly_work_packages)  %>
     <%= f.check_box :is_readonly, disabled: disabled %>
-    <div class="form--field-instructions">
-      <p><%= t('statuses.edit.status_readonly_html') %></p>
-    </div>
+
     <% if disabled %>
-      <div class="form--field-instructions -no-italic -xwide">
+      <div class="form--field-instructions -no-italic -wide">
```

Comment:
The class `-wide` does not exist for `form--field-instructions`, so it does not have any effect. Further, I chose `xwide` on purpose (and made it match the other breakpoints with that name) to avoid a line break in the buttons:
<img width="500" alt="Bildschirmfoto 2022-08-08 um 10 41 48" src="https://user-images.githubusercontent.com/7457313/183377558-b917c3ff-f43e-4212-b26d-8cf4d5982927.png">